workflow-ui/compose/src/main/java/com/squareup/workflow1/ui/compose/ViewEnvironmentWithComposeSupport.kt

```diff
@@ -1,11 +1,34 @@
 package com.squareup.workflow1.ui.compose
 
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.Modifier
 import com.squareup.workflow1.ui.Screen
 import com.squareup.workflow1.ui.ScreenViewFactory
 import com.squareup.workflow1.ui.ScreenViewFactoryFinder
 import com.squareup.workflow1.ui.ViewEnvironment
 import com.squareup.workflow1.ui.WorkflowUiExperimentalApi
 
+/**
+ * Alternative to [WorkflowLayout][com.squareup.workflow1.ui.WorkflowLayout]
+ * for a pure Compose application. Makes the receiver available via [LocalWorkflowEnvironment]
+ * and runs the composition bound to [screen].
+ *
+ * Note that any app relying on stock navigation classes like
+ * [BackStackScreen][com.squareup.workflow1.ui.navigation.BackStackScreen] or
+ * [BodyAndOverlaysScreen][com.squareup.workflow1.ui.navigation.BodyAndOverlaysScreen]
+ * are not pure Compose, and must call [ViewEnvironment.withComposeInteropSupport] first.
+ */
+@WorkflowUiExperimentalApi
+@Composable public fun ViewEnvironment.RootScreen(
```

Comment:
Nit: I'm bikeshedding but I think we should stick with either `Screen` or `Rendering` as a naming convention for these functions.

Reply:
Couldn't agree more. Should be `screen`.

Reply:
I should make a separate pass to catch old `rendering: Screen` instances :/